mongorestore/mongorestore_test.go

```diff
@@ -1749,10 +1749,16 @@ func TestRestoreTimeseriesCollections(t *testing.T) {
 		})
 
 		Convey("restoring a timeseries collection when the system.buckets collection already exists on the destination should fail", func() {
-			testdb.RunCommand(context.Background(), bson.M{"create": "system.buckets.foo_ts"})
-			args = append(args, DirectoryOption, "testdata/timeseries_tests/ts_dump")
 			restore, err := getRestoreWithArgs(args...)
 			So(err, ShouldBeNil)
+			// In the 8.0 release, this no longer leads to an error, so
+			// there's nothing to test here.
+			if restore.serverVersion.LT(db.Version{8, 0, 0}) {
```

Comment:
Doesn't the comparison mean 
if server version < 8.0, then 
     return
? 

I thought we were only going to test when the server version < 8.0.

Reply:
Oops, yeah, I got this backwards. I wonder if the pre-release is comparing as less than 8.0.0, which is why this passed. I guess I'll find out now that I fixed the comparison.